src/ebal.m

```diff
@@ -461,11 +461,11 @@
     Rntot_PAR       = LAI * (Fc * Rnh_PAR  + equations.meanleaf(canopy, Rnu_PAR, 'angles_and_layers', Ps)); % net PAR leaves
     aPAR_Cab_eta        = LAI * (Fc * (profiles.etah .* Rnh_PAR) + equations.meanleaf(canopy, profiles.etau .* Rnu_PAR, 'angles_and_layers', Ps));
     % ... green ePAR * relative fluorescence emission efficiency
-    %%%%%%%%%%%%%%%%%%% [Delta_Rltot] = Root_properties(Actot,rroot);
-    %%%%%%%%%%%%%%%%%%% Delta_Rl = fc*Delta_Rltot;
-    %%%%%%%%%%%%%%%%%%% Rl = Rl + Delta_Rl;
-    %%%%%%%%%%%%%%%%%%% Rltot = sum(sum(Rl));
-    %%%%%%%%%%%%%%%%%%% fc = Rl./Rltot;
+    % [Delta_Rltot] = Root_properties(Rl, Ac, rroot, frac, bbx, KT, DeltZ, sfactor, LAI_msr);
+    % Delta_Rl = fc*Delta_Rltot;
+    % Rl = Rl + Delta_Rl;
+    % Rltot = sum(sum(Rl));
+    % fc = Rl./Rltot;
```

Comment:
Why not deleting these lines of code? Are they kept for a reason?

Reply:
ok, I will delete them.

Reply:
If we delete these lines of code, the `Root_properties` will never be called, which means that we delete the root dynamic growth module in our model. Could we add an optional index to decide if we open the root dynamic simulation?